src/js/ajaxSlideshow.js

```diff
@@ -0,0 +1,53 @@
+$(document).ready(function() {
+  // $.ajaxSetup({
+  //   headers: {
+  //     "Access-Control-Allow-Origin": true
+  //   }
+  // });
+  //  create function to cycle through array of API data/img links and apply it to the #photo div
+  var count = 0;
+  var slideshow = function(pics) {
+    $('#show').attr('src', pics[count]);
+  };
+
+  //  clears the textfield when clicked on
+  $('#input').click(function(event) {
+    $('#input').val('');
+  });
+
+  //  creat click event for when click on submit
+  $('#search').click(function(event) {
+    event.preventDefault();
+    //  gets input value
+    var input = $('#input').val();
+    //  puts value in query
+    $.get('https://www.reddit.com/r/pics/search.json', {
+      q: input
+    }).always(function(data) {
```

Comment:
You may want to change this to `.done`, in case an error occurs when accessing Reddit (for example, Reddit's servers could be down)